### PR TITLE
Segregate S3client Procedure Sections for OVA 1.0.4

### PR DESCRIPTION
**Pull request by TechWriter-Mayur**
Using the AWS CLI procedure has been broke into 2 different procedure:

1. Procedure 1: To install the AWS CLI on the CORTX OVA to use for IO operations.
2. Procedure 2: To install the AWS CLI on the different VM to use for the IO operations.

Signed-off-by: TechWriter-Mayur <mayur.gupta@seagate.com>

### CODE REVIEW

File: doc/Performing_IO_Operations_Using_S3Client.rst [hunk 1]
```diff
@@ -4,6 +4,12 @@ Performing IO Operations Using S3 Client
 
 If you want to use the provided OVA to test CORTX and observe activity in its GUI, you should have already followed the `instructions <CORTX_on_Open_Virtual_Appliance.rst>`_ to install and configure the OVA.  If you have not yet done so, please do that before following the rest of the instructions on this page.
 
+To perform the IO operations in CORTX, you need to install the AWS CLI. You can install the AWS CLI either on the same OVA or you can create a new VM and install it.
+
```
Comment (mukul-seagate11) [inline doc/Performing_IO_Operations_Using_S3Client.rst:8]: 's3client' should be mentioned

File: doc/Performing_IO_Operations_Using_S3Client.rst [hunk 2]
```diff
@@ -4,6 +4,12 @@ Performing IO Operations Using S3 Client
...
+For more information on:
+
+-  AWS CLI on the same OVA, see `AWS CLI on the CORTX OVA VM <#AWS-CLI-on-the-CORTX-OVA-VM>`__.
+-  AWS CLI on the new VM, see `AWS CLI on a different VM <#AWS-CLI-on-a-different-VM>`__.
```
Comment (mukul-seagate11) [inline doc/Performing_IO_Operations_Using_S3Client.rst:13]: 's3client' should be mentioned as "AWS CLI on same OVA as client"

Comment (TechWriter-Mayur) [reply doc/Performing_IO_Operations_Using_S3Client.rst:13]: As discussed, this is not required.

**Review by mukul-seagate11 — APPROVED** [review]
looks good

**Comment by mukul-seagate11** [comment]
@AshwiniS-Borse, please review the PR from QA perspective?

**Review by AshwiniS-Borse — APPROVED** [review]

**Pull request merged**